lib/doctor/Doctor.ts

```diff
@@ -112,6 +116,26 @@ export class Doctor {
     return [this.checkIfAccessTokenValid()];
   }
 
+  private performDefaultAccountOverrideFileChecks(): Array<Promise<void>> {
```

Comment:
I was a little confused why this has to return an empty array and it looks like the other functions in this file return `Array<Promise>` because they're all doing something asynchronous.  Returning the arrays of promises lets them be used up [here in the promise.all()](https://github.com/HubSpot/hubspot-cli/pull/1349/files#diff-e4ba2d37078a484aad98f6bb4d2e0f34fbf3d6e4c9d1ced40dec710ba9a55775R67-R72). This function isn't doing anything asynchronous so it doesn't need to follow that pattern. This function can have a return type of `void` and it can be removed from the promise.all chain above.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1349/commits/5cdb2178041106ba8f1ea6733e535a3c005628e4